modules/exploits/windows/local/cve_2020_0668_service_tracing.rb

```diff
@@ -24,61 +23,63 @@ class MetasploitModule < Msf::Exploit::Local
   include Msf::Post::File
 
   def initialize(info = {})
-    super(update_info(info,
-                      'Name'           => 'Service Tracing Privilege Elevation Vulnerability',
-                      'Description'    => %q(This module leverages a
```

Comment:
@adfoster-r7: Looks like RuboCop can't handle the first and last lines in this description, due to the chosen formatting. See https://github.com/rapid7/metasploit-framework/pull/13585/commits/e669d5d78d066adf2c3a6c14751e7f5db855334f.

Reply:
I believe this is the problematic code:

https://github.com/rapid7/metasploit-framework/blob/1bad53ae9884dbea2c43558eec9c03fb4edf5e24/lib/rubocop/cop/layout/module_description_indentation.rb#L54

Reply:
Good spot 👀 

Bug fix - https://github.com/rapid7/metasploit-framework/pull/13589